src/OpenSage.Mods.Generals/Gui/MainMenuCallbacks.cs

```diff
@@ -140,6 +142,13 @@ void OpenSinglePlayerSideMenu(string side, string sideWindowSuffix)
                             context.WindowManager.SetWindow(@"Menus\LanLobbyMenu.wnd");
                             break;
 
+                        case "MainMenu.wnd:ButtonOnline":
+                            // this should load another window, but we don't have a lobby server yet,
+                            // so we just show the "Direct Connect" window
+
+                            context.WindowManager.SetWindow(@"Menus\NetworkDirectConnect.wnd", NetworkUtils.OnlineTag);
```

Comment:
We need to keep in mind that this. Needs to be changed. Maybe add a "TODO:" , so it's easier to find later on?